test/extended/deployments/deployments.go

```diff
@@ -298,7 +298,14 @@ var _ = g.Describe("deploymentconfigs", func() {
 			_, name, err := createFixture(oc, envRefDeploymentFixture)
 			o.Expect(err).NotTo(o.HaveOccurred())
 
-			out, err := oc.Run("deploy").Args("dc/"+name, "--latest", "--follow").Output()
+			_, err := oc.Run("rollout").Args("latest", name).Output()
```

Comment:
@kargakis so to prevent conflicts I should wait for the observed generations to wait? or is there a better way to wait?